_common/README

```diff
@@ -0,0 +1,38 @@
+<!-- README Template for ory/examples. -->
+<!-- Use the following template for all READMEs in ory/examples. -->
+
+# Title 
+
+<!--Add one sentence describing the project, usecase and key services used. -->
+<!--Add a link to post on the Ory Blog as well as other platforms (dev.to, ...). -->
+
+## Overview
+
+<!-- Optional: Add an diagram if using multiple services or a more detailed explanation than in description or links to other relevant resources. -->
+
+### Prerequisites
```

Comment:
I could see the prerequisites making more sense as a subsection under `## Develop` than under `## Overview`.